src/canvascontext.ts

```diff
@@ -111,6 +113,17 @@ export class CanvasContext extends RenderContext {
     // Containers not implemented.
   }
 
+  openRotation(angle: number, x: number, y: number) {
```

Comment:
Maybe we should rename angle to `angleDegrees` to be more explicit.

Reply:
Or should we instead accept radians? Then calls to openRotation() below can pass in `Math.PI * 0.5`...?

Reply:
Changed to `angleDegrees`. I decided to go for degrees because this is what SVG uses in rotate.